backend/TreeOfAKind.Application/Command/Trees/AddPerson/AddPersonCommand.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using TreeOfAKind.Domain.Trees;
+using TreeOfAKind.Domain.Trees.People;
+
+namespace TreeOfAKind.Application.Command.Trees.AddPerson
+{
+    public class AddPersonCommand : TreeOperationCommandBase<PersonId>
```

Comment:
That would be more convenient if it would also take a list of relations to add.

Reply:
True thet,
I forgot

Reply:
Comment whether you like the updated AddPerson API